lib/pry/commands/clear_screen.rb

```diff
@@ -0,0 +1,14 @@
+class Pry::Command::ClearScreen < Pry::ClassCommand
+  match 'clear-screen'
+  group 'Input and Output'
+  description 'Clear the contents of the screen/window Pry is running in.'
+
+  def process
+    if windows?
+      _pry_.config.system.call(_pry_.output, 'cls', _pry_)
```

Comment:
i'm not familiar with this API, when was this introduced? it looks super complicated!! why not just do: `system("cls")` ?

Reply:
[6 years ago]( https://github.com/pry/pry/commit/66c464bafe2bfabcce510db26493220b34b83f13).
by default it handles the command returning a non successful exit code but you're right its argument list is quite complex. it handles errors, though, so that's why we use it otherwise we have to do it ourselves.